### DISCUSSION TITLE
Welcome to WrongSecrets Discussions!

### DISCUSSION THREAD

**Discussion by commjoen** [OWASP/wrongsecrets#59] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community, next to using [OWASP Slack](https://owasp.slack.com/archives/C02KQ7D9XHR)[sign up](https://owasp.org/slack/invite). We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas on secret challenges and/or detection or management tools
  * Engage with other community members.

**Comment by vishakha-mali** [comment]
Is the Slack channel available for official members only, because I couldn't sign-in? If yes , apologies and if no then I have to make own email according to owasp.org??

**Reply by commjoen** [reply]
Hi @vishakha-mali , you can sign up via https://owasp.org/slack/invite : put your email in so you can get an invite to the Slack workspace and get in touch with us 👍
